src/components/labwareResult/LabwareResult.tsx

```diff
@@ -180,7 +183,7 @@ export default function LabwareResult({
                   <FailIcon
```

Comment:
is there a reason for replacing the non-null assertion by the optional chaining for the other fields here (other than result)

Reply:
Yes, result will become an optiona field on graphql request(core) with these new changes

Reply:
This is to avoid the situation where the component fails to load  or throws exception in case these fields are missing especially while calling from unit tests